### ISSUE TITLE
Enable plurals tags to match android format

### ISSUE THREAD

**Issue by davidec-twinlogix** [icerockdev/moko-resources#145]
I'm using moko-resources in a project where resource files are pulled from an external translation service in android format.
It would be nice that also "plurals" tag is processed along with "plural".

```
 <plurals name="hours">
        <item quantity="one">%1$s hour</item>
        <item quantity="other">%1$s hours</item>
 </plurals>

 <plural name="weeks">
        <item quantity="one">%1$s week</item>
        <item quantity="other">%1$s weeks</item>
 </plural>
```

**Comment by steviek** [comment]
I just ran into the same issue earlier, I thought it wasn't working and didn't realize it was `plural` instead of `plurals` until I copy+pasted the example code.

If changing to `plurals` isn't possible, having the generator throw an exception when you use `plurals` would be helpful to debug issues

**Comment by Alex009** [comment]
i think we can add alias for plural in generator.
and maybe we should add warnings in parsing of xml when found unknown tags

**Comment by Alex009** [comment]
will be available in 0.18.0.
thanks to @asapha
